### PR TITLE
rewrite: Chinese translation

### PR DESCRIPTION
**Pull request by SakuraSa233**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ x ] Acknowledge that you're contributing your code under Apache v2.0 license

- [ - ] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

I think that the former Chinese translation is no so good. E.g. he translated "longitude" as "length" just because "longitude" is written as "Long"? Maybe machine translation?

If you updated something else, you could contact me do complete its translation.

### CODE REVIEW

**Comment by liuyunli** [comment]
i will check later

File: GPSTest/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 */
 -->
 <resources>
-    <string name="app_name">GPS测试</string>
+    <string name="app_name">GPSTest</string>
```
Comment (liuyunli) [inline GPSTest/src/main/res/values-zh-rCN/strings.xml]: GSP工具  may better

Comment (liuyunli) [inline GPSTest/src/main/res/values-zh-rCN/strings.xml]:
```suggestion
     <string name="app_name">GPS工具</string>
```

File: GPSTest/src/main/res/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -131,194 +140,214 @@
...
+        - 请见 http://bit.ly/gpstest-releases
     </string>
     <string name="main_help_close">关闭</string>
 
     <!-- Preferences -->
     <string name="pref_title">设置</string>
     <string name="pref_map_category_title">地图</string>
     <string name="pref_map_type_title">地图类型</string>
-    <string name="pref_map_type_summary">设置地图展示类型</string>
+    <string name="pref_map_type_summary">选择现实的地图类型</string>
```
Comment (liuyunli) [inline GPSTest/src/main/res/values-zh-rCN/strings.xml]:
```suggestion
     <string name="pref_map_type_summary">选择显示的地图类型</string>
```

File: GPSTest/src/main/res/values-zh-rCN/strings.xml [hunk 3]
```diff
@@ -131,194 +140,214 @@
...
     <string name="pref_map_type_dialog_title">选择地图类型</string>
 
-    <string name="pref_rotate_map_with_compass_title">通过司南旋转地图</string>
-    <string name="pref_rotate_map_with_compass_summary">根据你的方向改变地图方向
-    </string>
+    <string name="pref_rotate_map_with_compass_title">旋转地图</string>
```
Comment (liuyunli) [inline GPSTest/src/main/res/values-zh-rCN/strings.xml]:
```suggestion
     <string name="pref_rotate_map_with_compass_title">旋转跟随</string>
```

File: GPSTest/src/main/res/values-zh-rCN/strings.xml [hunk 4]
```diff
@@ -131,194 +140,214 @@
...
+    <string name="pref_rotate_map_with_compass_summary">根据你的方向来改变地图朝向</string>
 
-    <string name="pref_tilt_map_with_sensors_title">通过设备倾斜地图</string>
-    <string name="pref_tilt_map_with_sensors_summary">举起设备时更改为第一人称视图
-    </string>
+    <string name="pref_tilt_map_with_sensors_title">倾斜地图</string>
```
Comment (liuyunli) [inline GPSTest/src/main/res/values-zh-rCN/strings.xml]:
```suggestion
     <string name="pref_tilt_map_with_sensors_title">倾斜跟随</string>
```

File: GPSTest/src/main/res/values-zh-rCN/strings.xml [hunk 5]
```diff
@@ -131,194 +140,214 @@
...
-    <string name="pref_gps_min_time_invalid_entry">区间必须是个数字</string>
+    <string name="pref_gps_min_time_title">时间间隔</string>
+    <string name="pref_gps_min_time_summary">GNSS 修正的时间间隔（单位：秒）</string>
+    <string name="pref_gps_min_time_dialog_title">输入时间间隔（单位：秒）</string>
+    <string name="pref_gps_min_time_invalid_entry">时间间隔必须是一个数字</string>
 
     <string name="pref_use_gnss_apis_title">使用GNSS APIs</string>
-    <string name="pref_use_gnss_apis_summary">可在Android 7或更高版本找到。如果未经检查，将使用遗留的GPS状态侦听器GPSStatus.Listener。
-    </string>
+    <string name="pref_use_gnss_apis_summary">在Android 7.0以及以上可用。取消选择后将会使用旧的GPSStatus.Listener。</string>
```
Comment (liuyunli) [inline GPSTest/src/main/res/values-zh-rCN/strings.xml]:
```suggestion
    <string name="pref_use_gnss_apis_summary">在Android 7.0或更高版本以上可用。取消选择后将会使用旧的GPSStatus.Listener。</string>
```

File: GPSTest/src/main/res/values-zh-rCN/strings.xml (hunk 1 above)
Comment (SakuraSa233) [reply GPSTest/src/main/res/values-zh-rCN/strings.xml]: I have said in the commit. I think keep it "GPSTest" is better.

File: GPSTest/src/main/res/values-zh-rCN/strings.xml (hunk 5 above)
Comment (SakuraSa233) [reply GPSTest/src/main/res/values-zh-rCN/strings.xml]: You had a Chinese grammar mistake here.
"更高" and "以上" are present the same meaning of higher.

But I'll try to improve the expression here later.

**Comment by SakuraSa233** [comment]
@liuyunli @wangkf 
I have fixed some mistakes in `b783595`.
Please check it.

**Comment by oldherl** [comment]
信噪比 is the standard translation of SNR, signal to noise ratio. It should not be changed back to English.

**Comment by SakuraSa233** [comment]
@oldherl It doesn't effect the whole. I could change it in next version.

**Comment by Skimige** [comment]
I've made another version at #245, how about take a look at it?

**Comment by barbeau** [comment]
I'm going to go ahead and merge this so everyone can work on the master branch.

@Skimige if you have any improvements to this translation please open in a new PR. Thanks!

**Comment by Skimige** [comment]
@barbeau All right. I may do so after you release this version of translation.

**Comment by barbeau** [comment]
@Skimige @oldherl @SakuraSa233 This translation is now available in beta v3.2.9.  Please take a look!

**Comment by barbeau** [comment]
Thanks @Skimige! Your suggestions make sense to me. Could you please put them in a pull request?

**Pull request merged**